### PR TITLE
Update Github Actions doc based on recent testing by Julian

### PR DESCRIPTION
**Pull request by Julian88Tex**
# Critical Changes

# Changes
- add recommended Github Personal Access Token scope with screenshot
- add name, on, and env code for Packaging org Github Action example

# Issues Closed

### CODE REVIEW

**Comment by Julian88Tex** [comment]
@Br4nd0R ready for review :-)

**Comment by prescod** [comment]
@Julian88Tex Thanks so much!

@jstvz Have you changed anything that would affect this?

**Comment by jstvz** [comment]
> @jstvz Have you changed anything that would affect this?

Yes. There'll be additional information in `cci service info github`:

```console
❯ cci service info github test-no-gist-scope
┌github:test-no-gist-scope───────────────────────────────────────────────────────────────────┐
│ Key           │ Value                                                                      │
├───────────────┼────────────────────────────────────────────────────────────────────────────┤
│ username      │ jstvz                                                                      │
├───────────────┼────────────────────────────────────────────────────────────────────────────┤
│ token         │ ghp_HQEAπππππππππππππππππππππππππiwiwTQ6                                   │
├───────────────┼────────────────────────────────────────────────────────────────────────────┤
│ email         │ jestevez@salesforce.com                                                    │
├───────────────┼────────────────────────────────────────────────────────────────────────────┤
│ Organizations │ salesforce, forcedotcom, SalesforceFoundation, roundcorner-products, SFDO- │
│               │ Tooling, SFDO-Community                                                    │
├───────────────┼────────────────────────────────────────────────────────────────────────────┤
│ scopes        │ {'read:user', 'repo', 'read:org'}                                          │
├───────────────┼────────────────────────────────────────────────────────────────────────────┤
│ SSO Disabled  │ salesforce, forcedotcom, SalesforceFoundation, SFDO-Tooling                │
└───────────────┴────────────────────────────────────────────────────────────────────────────┘
```

**Comment by davisagli** [comment]
Just a heads up that we are about to merge #2676 which reorganizes these docs a bit. @Br4nd0R could you help update this once we merge that PR?

File: docs/github_actions.rst [hunk 1]
```diff
@@ -374,6 +377,23 @@ in our demo repository.
...
+   on:
+      push:
+         branches:
+            - master
+         paths-ignore:
+            - 'docs/**'
+            - 'README.md'
+
+   env:
+      CUMULUSCI_KEYCHAIN_CLASS: cumulusci.core.keychain.EnvironmentProjectKeychain
```
Comment (Br4nd0R) [inline docs/github_actions.rst:391]: This is no longer needed as of merge that took place earlier today.
```suggestion
```

**Review by Br4nd0R — CHANGES_REQUESTED** [review]
Thanks for the contribution @Julian88Tex - Just the one line that should be removed. All else looks good!

**Comment by Br4nd0R** [comment]
> Just a heads up that we are about to merge #2676 which reorganizes these docs a bit. @Br4nd0R could you help update this once we merge that PR?

@davisagli - It looks like the contribution points here still fit nicely where they are (even with the addition of the new `headless.rst` section). The first part is going over the proper scopes to assign, and the second is an improvement to the existing github workflow example. I think both are still relevant to `github_actions.rst`.

**Review by Br4nd0R — APPROVED** [review]

File: docs/github_actions.rst (hunk 1 above)
Comment (Julian88Tex) [reply docs/github_actions.rst]: @Br4nd0R thanks for the change and the merge :)

**Pull request merged**